### ISSUE TITLE
failed to install

### ISSUE THREAD

**Issue by windevilhshs** [Aura-healthcare/ecg_qc#15]
hello, im learning your library. I'm facing a problem now and I'm feeling so bad. after running pip install ecg-qc, I cannot import it. 
the error says: 
import ecg_qc
Traceback (most recent call last):
  File "<ipython-input-61-b9b6516df03c>", line 1, in <module>
    import ecg_qc
ModuleNotFoundError: No module named 'ecg_qc'

so I find the folder, there's no folder 'ecg_qc' but just a folder 'ecg_qc-1.0b6.dist-info'. So I tried the next way, run '$ git clone https://github.com/Aura-healthcare/ecg_qc.git
$ python setup.py install' after some errors and solutions, the terminal says 'Finished processing dependencies for ecg-qc==1.0b6'. I was happy to see it, there is a 'ecg_qc-1.0b6-py3.8.egg'  near the folder 'ecg_qc-1.0b6.dist-info'. im happy to see it.
BUT, when I try to run 'ecg_qc = EcgQc(model='rfc_norm_2s.pkl',
               sampling_frequency=256,
               normalized=True)'
it says: 'ecg_qc = EcgQc(model='rfc_norm_2s.pkl',
TypeError: __init__() got an unexpected keyword argument 'model''
so I delete the model, and run the rest of the sentence.
but It says, 'FileNotFoundError: [Errno 2] No such file or directory: 'rfc_norm_2s.pkl''
if I download the folder'ecg_qc' from github and paste it on the correct path, there's still the same error.
so I guess there's still something wrong with the installing.
I tried both on Mac and windows, all failed.
I'm so confused. what should I do? how to install the package successfully? please tell me, thank you very much.

**Comment by windevilhshs** [comment]
hello author, I find that the function should be sqi_scores = ecg_qc.compute_sqi_scores(signal),not  sqi_scores = ecg_qc.compute_sqi_score(signal).
and also 
`ecg_qc = EcgQc('rfc_norm_2s.pkl',
               sampling_frequency=fs,
               normalized=True)
`
don't have to write 'model =' before the model name.
I'm ok now, I don't have other questions. I just hope you can fix the pip install problem, so that will not affect others to use it.
Finally, I want to say, thank you for your efforts. My English is not very good. Please forgive me.

**Comment by Roneival** [comment]
i totally agree with you, had to work around the functions to get a result ! 
I  guess that pip install is now fixed :)

**Comment by adgilbert** [comment]
I had the same issues... you have to use `model_file` when calling EcgQc.

I tried to fix the install issue in https://github.com/Aura-healthcare/ecg_qc/pull/16

**Comment by rdyan0053** [comment]
I have same error:
ImportError: cannot import name 'EcgQc' from 'ecg_qc'

But I had install the ecg_qc by using the `pip install ecg-qc` on Linux server
